src/routes/Home.js

```diff
@@ -1,11 +1,44 @@
 import React from 'react';
 
 import OrganizationsList from '../components/OrganizationsList';
+import Input from '../ui-kit/components/Input';
+import Button from '../ui-kit/components/Button';
+
+import styled from 'styled-components';
```

Comment:
styled-components import should go right after React since it's a package, not our custom component or module

Reply:
We should add this rule to our eslint config

Reply:
yeah, planning to do that (see comments in https://wannabook.atlassian.net/browse/WNB-65)